dash/src/util/LocalityDomain.cc

```diff
@@ -81,18 +75,20 @@ dash::util::LocalityDomain::LocalityDomain(
 
 dash::util::LocalityDomain::~LocalityDomain()
 {
-  DASH_LOG_TRACE_VAR("LocalityDomain.~()", _domain->domain_tag);
+  DASH_LOG_TRACE_VAR("LocalityDomain.~()", _domain);
 
-  if (_subdomains != nullptr) {
+  if (nullptr != _subdomains) {
```

Comment:
Yup!
But hey, the Yoda Condition is actually good style in comparison operations if one of the operands is const. It avoids the common kind of typo `if (important_pointer = nullptr)`.